socs/agents/orientalmotor_blh/agent.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python3
+'''OCS agent for BLH motor driver
+'''
+import argparse
+import os
+import time
+
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import TimeoutLock
+
+from socs.agents.orientalmotor_blh.drivers import BLH
+
+PORT_DEFAULT = '/dev/ttyACM0'
+LOCK_RELEASE_SEC = 1.
+LOCK_RELEASE_TIMEOUT = 10
+ACQ_TIMEOUT = 100
+INIT_TIMEOUT = 100
+
+
+class BLHAgent:
+    '''OCS agent class for BLH motor driver
+    '''
+
+    def __init__(self, agent, port=PORT_DEFAULT):
+        '''
+        Parameters
+        ----------
+        port : string
+            Port to connect
+        '''
+        self.active = True
+        self.agent = agent
+        self.log = agent.log
+        self.lock = TimeoutLock()
+        self.take_data = False
+
+        self._blh = BLH(port=port)
+
+        self.initialized = False
+
+        agg_params = {'frame_length': 60}
+        self.agent.register_feed('motor',
+                                 record=True,
+                                 agg_params=agg_params,
+                                 buffer_time=1)
+        self.speed = 0.0
+
+    def init_blh(self, session, params=None):
+        '''Initialization of BLH motor driver'''
+        if self.initialized:
+            return True, 'Already initialized'
+
+        with self.lock.acquire_timeout(0, job='init_blh') as acquired:
+            if not acquired:
+                self.log.warn('Could not start init because '
+                              '{} is already running'.format(self.lock.job))
+                return False, 'Could not acquire lock.'
+
+            session.set_status('starting')
+
+            self._blh.connect()
+            session.add_message('BLH initialized.')
+
+        self.initialized = True
+
+        return True, 'BLH module initialized.'
+
+    def start_acq(self, session, params):
+        '''Starts acquiring data.
+        '''
```

Comment:
Please standardize the naming and docstrings according to https://ocs.readthedocs.io/en/main/developer/agent_references/documentation.html#task-and-process-documentation.